portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
```

Comment:
A suggestion for an new feature you could try: make it so that every time you click the button the fact changes (so that you never click the button and the fact randomly happens to be the same as the already displayed fact)

[Note about reviews] sometimes something like this will come up in a review, and you can choose either: incorporate the suggestion into this PR with an additional commit, or, if it is a feature that would make this PR too complicated or big, you can always feel free to just reply to the comment with something like "Will address in future PR" or something